web/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "perses",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^4.12.3",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.15",
+    "@types/react-dom": "^17.0.0",
```

Comment:
Hopefully the @types should also be part of the devDependencies.

Reply:
Ok yeah, `create-react-app` just doesn't care much about `devDependencies` and puts everything under `dependencies` by default, but will change.